elfcore/src/coredump.rs

```diff
@@ -1380,6 +1376,28 @@ fn write_mapped_files_note<T: Write>(
     Ok(written)
 }
 
+fn write_custom_notes<T: Write>(
+    writer: &mut ElfCoreWriter<T>,
+    pv: &ProcessView,
+) -> Result<usize, CoreError> {
+    let mut total_written = 0;
+
+    for (name, data) in &pv.custom_notes {
+        tracing::info!(
```

Comment:
I wonder if we should have this at info level or not - what do we do for the other tracing statements?